src/test/java/org/broadinstitute/hellbender/IntelDeflaterIntegrationTest.java

```diff
@@ -1,29 +1,129 @@
 package org.broadinstitute.hellbender;
 
+import com.intel.gkl.compression.IntelDeflaterFactory;
+import htsjdk.samtools.*;
 import htsjdk.samtools.util.zip.DeflaterFactory;
-import org.broadinstitute.hellbender.utils.NativeUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.utils.RandomDNA;
 import org.broadinstitute.hellbender.utils.test.BaseTest;
 import org.testng.Assert;
-import org.testng.SkipException;
 import org.testng.annotations.Test;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.zip.Deflater;
+import java.util.zip.Inflater;
+
 /**
  * Test that it's possible to load libIntelDeflater
  */
 public class IntelDeflaterIntegrationTest extends BaseTest {
 
-    @Test
-    public void testIntelDeflaterIsAvailable(){
-        if ( ! NativeUtils.runningOnLinux() ) {
-            throw new SkipException("IntelDeflater not available on this platform");
-        }
+    private final static Logger log = LogManager.getLogger(IntelDeflaterIntegrationTest.class);
+    private static final String publicTestDirRelative = "src/test/resources/";
+    private static final String publicTestDir = new File(gatkDirectory, publicTestDirRelative).getAbsolutePath() + "/";
+    private static final String INPUT_FILE = publicTestDir + "CEUTrio.HiSeq.WGS.b37.NA12878.20.21.bam";
 
-        if ( NativeUtils.runningOnPPCArchitecture() ) {
-            throw new SkipException("IntelDeflater not available for this architecture");
-        }
+    @Test(enabled = true)
+    public void loadIntelDeflater() {
```

Comment:
Can you restore the `SkipExceptions` when `! NativeUtils.runningOnLinux()` or `NativeUtils.runningOnPPCArchitecture()`? We want this test to be skipped on unsupported architectures/platforms.